manifests/ppa.pp

```diff
@@ -5,6 +5,8 @@
   $options        = $::apt::ppa_options,
   $package_name   = $::apt::ppa_package,
   $package_manage = false,
+  $proxy_host     = undef,
+  $proxy_port     = 8080,
```

Comment:
@daenney I don't hate that, but I am a little confused with your hash syntax. I've only remembering seeing puppet hashes with `=>` as the separator between key/value (and also [this](https://docs.puppetlabs.com/puppet/latest/reference/lang_datatypes.html#hashes)). Is this something new?